diff --git a/config.h b/config.h
index 19136a1..ddfb76b 100644
--- a/config.h
+++ b/config.h
@@ -105,6 +105,7 @@ static const enum libinput_config_tap_button_map button_map = LIBINPUT_CONFIG_TA
 
 /* commands */
 static const char *term[]            = { "footclient", NULL };
+static const char *lf[]              = { "footclient", "lf", NULL };
 static const char *tofi[]            = { "sh", "-c", "tofi-run | xargs -r sh -c", NULL };
 static const char *brave[]           = { "brave-origin", NULL };
 static const char *wlogoutcmd[]      = { "wlogout", "-b", "2", NULL };
@@ -121,7 +122,8 @@ static const char *screenshots[]     = { "/home/bren/.local/bin/screenshots", NU
 static const Key keys[] = {
 	/* modifier                  key                 function          argument */
 	{ MODKEY,                    XKB_KEY_Return,     spawn,            {.v = term} },
-	{ MODKEY,                    XKB_KEY_Tab,        spawn,            {.v = tofi} },
+    { MODKEY,                    XKB_KEY_e,          spawn,            {.v = lf} },
+    { MODKEY,                    XKB_KEY_Tab,        spawn,            {.v = tofi} },
 	{ MODKEY,                    XKB_KEY_b,          spawn,            {.v = brave} },
 	{ MODKEY,                    XKB_KEY_Escape,     spawn,            {.v = wlogoutcmd } },
     { MODKEY|WLR_MODIFIER_SHIFT, XKB_KEY_L,          spawn,            {.v = swaylockcmd } },
